client/my-sites/stats/activity-log/index.jsx

```diff
@@ -234,17 +234,17 @@ class ActivityLog extends Component {
 		const logsGroupedByDay = map(
 			groupBy(
 				logsForMonth,
-				log => applySiteOffset( moment.utc( log.ts_utc ) ).format( 'LL' )
+				log => applySiteOffset( moment.utc( log.ts_utc ) ).startOf( 'day' ).valueOf()
 			),
-			( daily_logs, day ) => (
+			( daily_logs, tsDayStart ) => (
 				<ActivityLogDay
 					allowRestore={ !! isPressable }
 					isRewindActive={ isRewindActive }
-					key={ day }
+					key={ tsDayStart }
 					logs={ daily_logs }
 					requestRestore={ this.handleRequestRestore }
 					siteId={ siteId }
-					day={ day }
+					tsDayStart={ +tsDayStart }
```

Comment:
Whenever we use numeric timestamps, it should be unix format to avoid confusion. This fails that test because it has been offset already.

This is borne out by the fact that we are offsetting this timestamp again [here](https://github.com/Automattic/wp-calypso/pull/15795/files#diff-7b013c6182282d2c9f90085ba311017eR38) and [here](https://github.com/Automattic/wp-calypso/pull/15795/files#diff-aee6796196bcce1fc52827c526d18fadR80).